mappings/remediation_advice.json

```diff
@@ -622,7 +622,7 @@
             },
             {
               "id": "on_password_change",
-              "remediation_advice": "Properly invalidate all sessions on the server-side when the user changes or resets their password. Additionally, invalidate all existing sessions on other devices.",
+              "remediation_advice": "Properly invalidate all user sessions server-side on password reset and at a minimum, invalidate all non-current user sessions sever-side on password change.",
```

Comment:
Is this saying once password reset is requested  the session should be invalidated, or upon change? Trying to de-tangle the two statements in this advice.